python/revng/restapi/schema.graphql.j2

```diff
@@ -0,0 +1,71 @@
+{#
```

Comment:
I think we use `tpl` for jinja2 templates.
If you think `j2` is more correct, please fix others.

Reply:
Officially Jinja suggests to [use the `.jinja` file extension](https://jinja.palletsprojects.com/en/3.0.x/templates/#template-file-extension)
`.j2` is one of the _de jure_ extensions of jinja templates (while I found no references for `.tpl` to be commonly used)
However it's probably best to be consistent and either keep `.tpl` or convert everything to `.jinja`

Reply:
Up to you, but let's be coherent, either one or the other.